app/services/search/school_search.rb

```diff
@@ -96,4 +97,23 @@ def apply_job_availability_filter(scope)
       scope.where.not(id: organisation_ids)
     end
   end
+
+  def apply_organisation_type_filter(scope)
+    return scope unless organisation_types.present?
+
+    establishment_code_filter = []
+    establishment_name_filter = []
+
+    if organisation_types.include?("Academy")
+      %w[10 11].each { |code| establishment_code_filter << code }
+      ["Academies", "Free Schools"].each { |name| establishment_name_filter << name }
```

Comment:
Same suggestion as above:

```suggestion
      establishment_code_filter.concat %w[10 11]
      establishment_name_filter.concat ["Academies", "Free Schools"]
```